sql/2023/01/web-fonts-usage.sql

```diff
@@ -0,0 +1,36 @@
+# HTTP Archive query to get % of WordPress sites using atleast one web font.
+#
+# WPP Research, Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/34
+SELECT
+  sp._TABLE_SUFFIX AS client,
+  COUNTIF(reqFont > 0) AS site_with_web_fonts,
+  COUNT(DISTINCT th.url) AS total,
+  COUNTIF(reqFont > 0) / COUNT(DISTINCT th.url) AS pct_site_with_web_fonts
```

Comment:
A few naming suggestion for consistency with other queries:
```suggestion
  COUNTIF(reqFont > 0) AS with_web_fonts,
  COUNT(DISTINCT th.url) AS total_wp_sites,
  COUNTIF(reqFont > 0) / COUNT(DISTINCT th.url) AS pct_with_web_fonts
```